src/App.js

```diff
@@ -17,7 +19,8 @@ function App() {
        <Route path="/cart" element={<Cart></Cart>}></Route>
```

Comment:
Here you don't need to use wrapper tag, You can directly put self closing tags, Like this `element={<Cart/>}`

Reply:
Thank you . Done